src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -678,6 +679,29 @@ struct am_counter
   }
 };
 
+struct xrt_smi_config
+{
+  using result_type = std::any;
+
+  static result_type
+  get(const xrt_core::device* device, key_type key, const std::any& reqType)
+  {
+    if (key != key_type::xrt_smi_config)
+      throw xrt_core::query::no_such_key(key, "Not implemented");
+    std::string xrt_smi_config;
```

Comment:
Blank line between single statement expression and following code.
```suggestion
    if (key != key_type::xrt_smi_config)
      throw xrt_core::query::no_such_key(key, "Not implemented");
      
    std::string xrt_smi_config;
```